deploy.sh

```diff
@@ -0,0 +1,10 @@
+docker pull quay.io/keboola/developer-portal-cli-v2:latest
```

Comment:
Nemáš nastavený exit při chybě!

Reply:
nerozumiem, myslis ten hashbang?

Reply:
no ted se ti spustej vsechny prikazy i kdyz prvni treba failne.

Reply:
chybí tam tohle https://github.com/keboola/darksky-augmentation/blob/58938953b8eff710c297d911b874a6e13338fb83/deploy.sh#L2

Reply:
aha jasne, ja som to zkopcil odtialto:
https://github.com/keboola/ex-snowflake-query-history/blob/master/deploy.sh
pridam to tam dik!